nightly/trigger-pipeline-cronjob.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: trigger-nightly-pipeline
+spec:
+  schedule: "0 5 * * *"
+  concurrencyPolicy: Forbid
+  startingDeadlineSeconds: 300
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: pipeline
+          containers:
+            - name: trigger-nightly-pipeline
+              image: curlimages/curl
+              args: ["curl", "-X", "POST", "--data", "{}", "el-nightly-cron-listener.kuadrant-pipelines.svc.cluster.local:8080"]
```

Comment:
This is dependent on the namespace, could we make it not so?

Reply:
I think so, let me try...